etna/transforms/decomposition/change_points_segmentation.py

```diff
@@ -0,0 +1,147 @@
+from typing import List
+from typing import Optional
+
+import pandas as pd
+from ruptures.base import BaseEstimator
+
+from etna.transforms.base import FutureMixin
+from etna.transforms.base import PerSegmentWrapper
+from etna.transforms.base import Transform
+from etna.transforms.decomposition.base_change_points import RupturesChangePointsModel
+from etna.transforms.decomposition.base_change_points import TTimestampInterval
+
+
+class _OneSegmentChangePointsSegmentationTransform(Transform):
+    """_OneSegmentChangePointsSegmentationTransform make label encoder to change points."""
+
+    def __init__(
+        self, in_column: str, out_column: str, change_point_model: BaseEstimator, **change_point_model_predict_params
+    ):
+        """Init _OneSegmentChangePointsSegmentationTransform.
+        Parameters
+        ----------
+        in_column:
+            name of column to apply transform to
+        out_column:
+            result column name. If not given use ``self.__repr__()``
+        change_point_model:
```

Comment:
Add "TODO" to replace this parameters with the instance of BaseChangePointsModelAdapter in ETNA 2.0